manifests/t/Typora/Typora/1.0.3/Typora.Typora.installer.yaml

```diff
@@ -1,66 +1,42 @@
-# Created with YamlCreate.ps1 v2.0.3 $debug=AUSU.7-2-0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
-
-PackageIdentifier: Typora.Typora
-PackageVersion: 1.0.3
-Platform:
-- Windows.Desktop
-MinimumOSVersion: 10.0.0.0
-InstallerType: inno
-InstallModes:
-- interactive
-- silent
-- silentWithProgress
-UpgradeBehavior: install
-FileExtensions:
-- apib
-- md
-- mdown
-- mdtext
-- mdtxt
-- mdwn
-- mdx
-- mkd
-- mmd
-- rmd
-- text
-- txt
-Installers:
-- Architecture: x64
-  Scope: user
-  InstallerUrl: https://typora.io/windows/typora-setup-x64.exe
-  InstallerSha256: AC11151E906F78F79294F6F7656AC67426AC0DE95A322EEFFFE15CBDE502206C
-  InstallerSwitches:
-    Custom: /CURRENTUSER
-- Architecture: x64
-  Scope: machine
-  InstallerUrl: https://typora.io/windows/typora-setup-x64.exe
-  InstallerSha256: AC11151E906F78F79294F6F7656AC67426AC0DE95A322EEFFFE15CBDE502206C
-  InstallerSwitches:
-    Custom: /ALLUSERS
-- Architecture: x86
-  Scope: user
-  InstallerUrl: https://typora.io/windows/typora-setup-ia32.exe
-  InstallerSha256: B8FEF97119001C768DF262B617D0EA636C45016854013963C213A15DCD948D93
-  InstallerSwitches:
-    Custom: /CURRENTUSER
-- Architecture: x86
-  Scope: machine
-  InstallerUrl: https://typora.io/windows/typora-setup-ia32.exe
-  InstallerSha256: B8FEF97119001C768DF262B617D0EA636C45016854013963C213A15DCD948D93
-  InstallerSwitches:
-    Custom: /ALLUSERS
-- Architecture: arm64
-  Scope: user
-  InstallerUrl: https://typora.io/windows/typora-setup-arm64.exe
-  InstallerSha256: 8824CFD2D114F75B1AD9764CF5FA4F50269BE25636E94D31BA84E9C6B966B82F
-  InstallerSwitches:
-    Custom: /CURRENTUSER
-- Architecture: arm64
-  Scope: machine
-  InstallerUrl: https://typora.io/windows/typora-setup-arm64.exe
-  InstallerSha256: 8824CFD2D114F75B1AD9764CF5FA4F50269BE25636E94D31BA84E9C6B966B82F
-  InstallerSwitches:
-    Custom: /ALLUSERS
-ManifestType: installer
-ManifestVersion: 1.1.0
+# Automatically updated by the winget bot at 2022/Jan/11
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Typora.Typora
+PackageVersion: 1.0.3
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+FileExtensions:
+- apib
+- md
+- mdown
+- mdtext
+- mdtxt
+- mdwn
+- mdx
+- mkd
+- mmd
+- rmd
+- text
+- txt
+Installers:
+- Architecture: arm64
+  Scope: user
+  InstallerUrl: https://typora.io/windows/typora-setup-arm64.exe

```

Comment:
Hmm, the arm64 version is not updated to 1.0.4 (same hash)
got updated to 1.0.4 in https://github.com/microsoft/winget-pkgs/pull/40460

Reply:
The way our automation works is per installer not per application. What happened here is that the x64 and x86 installers got updated to 1.0.4 but arm64 didn't hence #40460 and this PR.

In fact arm64 is not even 1.0.3 but 1.0.2. I think the easiest way is to let this through and then move to it 1.0.2

![image](https://user-images.githubusercontent.com/36937303/149016386-b5d787fb-e6cb-42f7-8c68-e2761ec5f1a0.png)

x64
![image](https://user-images.githubusercontent.com/36937303/149016540-bd4ca714-d111-4654-9811-b50736952de2.png)

x86
![image](https://user-images.githubusercontent.com/36937303/149016722-915767bb-6237-45c1-999a-078048bf1290.png)